myke-roly/kino
# T56 resolve conflicts (#10)

* tabs screens 🖲️

* styles global center

* Components: title and subtitle

* Component: Slider

* State: movies request ☯️

* print movies populars

* swipe movies

* remove console.log

* State: save movie 🔖

* refactor: state auth

* Firebase: get user data

* Screen: Movie 🎥

* State: get movie by Id

* Screen: Movie print data

* use firebase authentication

* fixed 🗃️

* Reestructured files 📁

* dep: Change firebase dependecie

* Components: Modal Search Filter and added Slider

* changed private repo status

* test color

* tesring

* modify readme

* configure prettier

* configure eslint

* configure eslint

* configure prettier

* remove file unnecesaries

* [T56] configure precommir check

* [T56] Add licence MIT

* [T56] Changed readme

Co-authored-by: mike <mike@underscope.io>
